### PR TITLE
Change PA calulcation for PolarizationCorrection

### LINKED ISSUE #14139: PolarizationCorrection algorithm is not producing the right results
It was brought to our attention by the instrument scientists using PolarizationCorrection, that is not performing the right calculations. A script has been supplied involving the correct calculations and is producing the right results for the users.

The plan is to change the current calculations we do in PolarizationCorrection in favour of the ones performed in the script.

After converting units for `OFFSPEC33767` (with no cropping) to Wavelength and executing the script that was supplied to us, we achieve this plot:
logy_plot_from_polarization_correct_script

This plot (with error bars) will be regarded as the desired result when repeating this process with PolarizationCorrection.

### PR DESCRIPTION
**Pull request by MattKing06**
The way that the current PolarizationCorrection algorithm was producing results was incorrect and the changes that have been made have been taken from a script that was doing the correction in the right way.

**To Test:**
- Load the [data](https://slack-files.com/T02J4MMEW-F0DLR0VJ9-0cb69392e9) into mantid
- Open both the [correction script](https://slack-files.com/T02J4MMEW-F0DLNTW20-64e6f05ad7) and the [executing script](https://slack-files.com/T02J4MMEW-F0DLVPD7C-04a2ec068c) in the Mantid script window

**Running Our algorithm**
- The units of the data have already been converted to wavelength so just run PolarizationCorrection algorithm with these values for the associated parameters in the dialog.:

InputWorkspace = 33767
PolarizationAnalysis = PA
CRho= 0.955384,0.021501,-0.002962,0.000112
CAlpha= 0.957789,0.019995,-0.002697,0.000099
CAp=  0.986906,-0.013945,0.002480,-0.000161 
CPp=  0.999517,-0.013878,0.001680,-0.000043
OutputWorkspace = anything that isn't the InputWorkspace name

*\* Running the script **
When both of the script have been loaded:
- Execute all of the code in `running_sci_script.py`
- Group all of the output workspaces (it will help when comparing)

**Comparing workspaces**
After we have output from both of these it should be a case of running `CheckWorkspacesMatch` on the two groups that we have after running the correction.

There should be no data-mismatches.

**Release Notes**
//TODO

Fixes #14139

### CODE REVIEW

**Comment by OwenArnold** [comment]
Didn't see any update to the rst documentation for this algorithm. Could you please add that.

**Comment by MattKing06** [comment]
I have made a separate issue #14229 for the documentation changes. However I can close that issue and bundle the doc changes into this PR.

**Comment by OwenArnold** [comment]
@MattKing06. No point of a separate issue. Just need to spend 3 minutes updating the existing existing documentation.

File: docs/source/algorithms/PolarizationCorrection-v1.rst [hunk 1]
```diff
@@ -26,6 +26,21 @@ Since this ratio is wavelength dependent, rho is a polynomial, which is expresse
 Since this ratio is wavelength dependent, alpha is a polynomial, which is expressed as a function of wavelength. For example:
 :math:`\alpha(\lambda) =\sum\limits_{i=0}^{i=2} K_{i}\centerdot\lambda^i`, can be provided as :math:`K_{0}, K_{1}, K_{2}`
 
+Output from Full Polarization Analysis 
+--------------------------------------
```
Comment (OwenArnold) [inline docs/source/algorithms/PolarizationCorrection-v1.rst]: Excellent!

**Pull request merged**